tests/cypress/template/concept.cy.js

```diff
@@ -95,6 +95,18 @@ describe('Concept page', () => {
     // check the concept prefLabel
     cy.get('#concept-heading h1').invoke('text').should('equal', 'music research')
   })
+  it('concept preflabel can be copied to clipboard', () => {
+    cy.visit('/yso/en/page/p21685') // go to "music research" concept page
+
+    // click the copy to clipboard button next to the prefLabel
+    cy.get('#copy-preflabel').click()
+
+    // check that the clipboard now contains "music research"
+    // NOTE: This test may fail when running Cypress interactively in a browser.
+    // The reason is browser security policies for accessing the clipboard.
+    // If that happens, make sure the browser window has focus and re-run the test.
```

Comment:
The things you learn..! -__-

Reply:
:grin: Ideally the test would just be skipped when this happens. Not sure if that's easy to implement.